ballot/migrations/0002_auto_20151219_0359.py

```diff
@@ -7,7 +7,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('ballot', '0002_load_ballot_data'),
+        ('ballot', '0001_initial'),
```

Comment:
Oh, this is clever. At first I was thinking you were going to break the migrations, but I guess since the load_ballot_data wasn't making any schema changes, it's safe to just remove that and patch the link.